horace_core/sqw/@sqw/private/binary_op_manager_single.m

```diff
@@ -10,6 +10,13 @@
 %           >> nd = dimensions(obj)
 %   (3) have private function that returns class name
 %           >> name = classname     % no argument - gets called by its association with the class
+%
+allowed_types = {'double', 'd0d', 'd1d', 'd2d', 'd3d', 'd4d', 'sqw', 'sigvar'};
```

Comment:
you test that `int64` is prohibited, but not that `int32`. Is `float` supported as `double` is?

Reply:
I've added cases for other non-double numeric types. The existing code only allowed `double` not any other numeric types (including `float`), so I've left as is. If we need to support `float` as well that should be trivial.